libbeat/logp/logger.go

```diff
@@ -96,6 +97,11 @@ func (l *Logger) DPanic(args ...interface{}) {
 	l.sugar.DPanic(args...)
 }
 
+// IsDebug checks to see if the given logger is Debug enabled.
+func (l *Logger) IsDebug() bool {
+	return l.sugar.Desugar().Check(zapcore.DebugLevel, "") != nil
```

Comment:
We should probably store the original zap.Logger inside of `logp.Logger`. Originally it held both, but I dropped it because it was never used.

Reply:
@andrewkroh, i have made the necessary change